whisper_experiments/tests/utils/test_string_utils.py

```diff
@@ -0,0 +1,193 @@
+import pytest
+from whisper_experiments.utils.string_utils import (
+    TextDiff,
+    Line,
+    AddedWord,
+    RemovedWord,
+    ModifiedWord,
+    LineComparison,
+    word_differences,
+    line_differences,
+)
+from text_diff import AddedLine, RemovedLine, ModifiedLine, Mask
+from typing import List
+
+AL = AddedLine
+RL = RemovedLine
+ML = ModifiedLine
+AW = AddedWord
+RW = RemovedWord
+MW = ModifiedWord
+T = TextDiff
+LC = LineComparison
+
+
+@pytest.mark.parametrize(
+    "text_diff_1, text_diff_2, equality",
+    [
```

Comment:
Can I just get a comment here explaining some of these tests. Like given in plain text what would a couple of these be?

Reply:
Great idea.